src/v2/Apps/Onboarding/Views/OnboardingQuestionTwo.tsx

```diff
@@ -27,9 +27,10 @@ import {
 import { useOnboardingContext } from "../useOnboardingContext"
 
 export const OnboardingQuestionTwo: FC = () => {
-  const { answers, setAnswers, setContext, next } = useOnboardingContext()
+  const { answers, setAnswerTwo, setBasis, next } = useOnboardingContext()
```

Comment:
Don't mean to drag this on, but what does "basis" mean? (Trying to find a way to see this function and understand what it's doing without much outside input.)

Reply:
This object is what's needed to make decisions to move to the conditional steps.

![](https://static.damonzucconi.com/_capture/DQzWcSKs.png)

from https://www.youtube.com/watch?v=V6DKjEbdYos